config/environment_config.yml

```diff
@@ -1,6 +1,7 @@
 defaults: &defaults
   admin_username: admin
   admin_password: password
+  admin_email: admin@example.com
```

Comment:
Is this supposed to be here or is it from merge ?

Reply:
Yes i need this to validate the acas download